go/vt/vtgate/engine/insert.go

```diff
@@ -363,6 +366,19 @@ func (ins *Insert) description() PrimitiveDescription {
 		other["VindexValues"] = valuesOffsets
 	}
 
+	// This is a check to ensure we send the correct query to the database.
+	// "ActualQuery" should not be part of the plan output, if it does, it means the query was not rewritten correctly.
+	if ins.Mid != nil {
+		var mids []string
+		for _, n := range ins.Mid {
+			mids = append(mids, sqlparser.String(n))
+		}
+		shardedQuery := ins.Prefix + strings.Join(mids, ", ") + ins.Alias + sqlparser.String(ins.Suffix)
+		if shardedQuery != ins.Query {
+			other["ActualQuery"] = shardedQuery
+		}
+	}
+
```

Comment:
Was this code intended for debugging to see when the sharded query and the original query are different?

Reply:
I kept this for a reason to fail in future if anything goes wrong in the final query output.